Makefile

```diff
@@ -91,7 +91,7 @@ $(TEMPDIR):
 
 .PHONY: bootstrap-tools
 bootstrap-tools: $(TEMPDIR)
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(TEMPDIR)/ v1.45.0
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(TEMPDIR)/ v1.46.2
```

Comment:
I have to bump to `1.45.2` or higher to support 1.18 as I used generics in my PR. Only `bodyclose`, `rowserrcheck`, `structcheck`, and `unparam` are still disabled in this release due to 1.18